src/Files/UserControls/InnerNavigationToolbar.xaml

```diff
@@ -205,6 +205,27 @@
                         <FontIcon FontFamily="{StaticResource RecycleBinIcons}" Glyph="&#xEF88;" />
                     </AppBarButton.Icon>
                 </AppBarButton>
+                <AppBarSeparator x:Name="ExtractButtonSeparator" Visibility="{x:Bind ViewModel.CanExtract, Mode=OneWay, FallbackValue=Collapsed}" />
+                <AppBarButton
+                    x:Name="ExtractButton"
+                    Width="Auto"
+                    MinWidth="40"
+                    Command="{x:Bind ViewModel.ExtractCommand, Mode=OneWay}"
+                    IsEnabled="{x:Bind ViewModel.CanExtract, Mode=OneWay, FallbackValue=False}"
+                    Label="{x:Bind GetLocalizedString('BaseLayoutItemContextFlyoutExtract/Text')}"
+                    LabelPosition="Default"
+                    Visibility="{x:Bind ViewModel.CanExtract, Mode=OneWay, FallbackValue=Collapsed}"
+                    KeyboardAcceleratorTextOverride="Ctrl+E">
```

Comment:
Do you want to add a resource string for this?

Reply:
Sure I can do that. Just want make sure I'm doing it correct though. I'll go through each resource file and add a "Extract" string to each and change the xaml to be 
```xml 
Label="{helpers:ResourceString Name=Extract}" 
```

Reply:
That's correct, if you're up to it, you can remove duplicate strings for "Extract" and use the new one instead.

Reply:
Excellent, that sounds like something I can do. I’ll get on it.

Reply:
You can also add a resource string called `ExtractKeyboardAcceleratorTextOverride`